pkg/commands/logging/bigquery/bigquery_integration_test.go

```diff
@@ -366,7 +370,7 @@ func getBigQueryError(i *fastly.GetBigQueryInput) (*fastly.BigQuery, error) {
 	return nil, errTest
 }
 
-var describeBigQueryOutput = "\n" + strings.TrimSpace(`
+var describeBigQueryOutput = "\n" + strings.TrimSpace(`Account name: `+`
```

Comment:
Yes and no- depending on your editor it removes the space after account name and then the test fails- this is a hacky way around it- the other way is to set it to an actual value so that it isn't a blank line./

Reply:
Fixed in https://github.com/fastly/cli/pull/699/commits/4e0a709927d48967447c7dc3fac676633ed339d9